libs/ledger-live-common/src/families/cosmos/js-synchronisation.ts

```diff
@@ -221,6 +229,7 @@ export const getAccountShape: GetAccountShape = async info => {
       pendingRewardsBalance,
       unbondingBalance,
       withdrawAddress,
+      sequence: accountInfo?.sequence ?? 0,
```

Comment:
I think a fallback to a 0 value isn't necessarily what we want, it may cause bugs.
Fortunately I don't think `accountInfo` or `sequence` can be undefined?
```suggestion
      sequence: accountInfo.sequence,
```

Reply:
fixed